src/dev.html

```diff
@@ -18,6 +18,8 @@
 
     <header id="site-header"></header>
 
+    <mkt-nav role="navigation" id="site-nav"></mkt-nav>
```

Comment:
I'm pretty sure this design isn't possible with the menu as a child of the header.

We could always inject the template after `#site-header`, but we should probably have the flexibility to use different templates as appropriate; this existing system is rigid and inflexible.